articles/quickstart.md

```diff
@@ -439,66 +472,36 @@ If we run this, we'll get exactly the same 50-50 result we got before. However,
             Set(Zero, q1);
         }
 
-        // Return number of times we saw a |0> and number of times we saw a |1>
+        // Return number of times we saw a |0> and number of times we saw a |1
+        Message("Test results (# of 0s, # of 1s, # of times agreed)")
         return (count-numOnes, numOnes, agree);
     }
 ```
 
-The new return value (`agree`) keeps track of every time the measurement from the first qubit matches the measurement of the second qubit. We also have to update the host application accordingly:
-
-#### [Python](#tab/tabid-python)
-
-```python
-import qsharp
+The new return value (`agree`) keeps track of every time the measurement from
+the first qubit matches the measurement of the second qubit.
 
-from qsharp import Result
-from Quantum.Bell import TestBellState
+Running the code we obtain:
 
-initials = {Result.Zero, Result.One} 
+```dotnetcli
+dotnet run --count 1000 --initial One
+(505, 495, 1000)
```

Comment:
```suggestion
Test results (# of 0s, # of 1s, # of agreements)
(505, 495, 1000)
```
Forgot to update the return to include the `Message` printing. If don't take re-phrasing suggestion above this would still have to be added, but simply using `Test results (# of 0s, # of 1s, # of times agreed)`

Reply:
Same green issue as above will appear when adding this output too though.